creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -126,29 +123,6 @@ const healthCheckController = async (req) => {
   return successResponse(response)
 }
 
-/**
```

Comment:
this route hasn't been working since those queues have not been running, and we don't use the route anywhere, safe to remove

Reply:
forgot about this. plus bull dashboard is so much better